tests/desktop_chrome.pm

```diff
@@ -0,0 +1,52 @@
+use base 'installedtest';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    check_desktop_clean();
+
+    # Run Chrome and navigate to a web page.
+    type_very_safely("chrome\n");
+
+    # Chrome isn’t installed by default; make sure it can be. When we launch
+    # the Chrome .desktop file from the shell, we’ll land straight in
+    # gnome-software which starts the installation process automatically and
```

Comment:
I *think* that what's actually happening here is that the installation starts automatically (on free images) when you log in, not when you launch Software, but I could be wrong.

Reply:
I was guessing. I’ll update the comment.

Reply:
fixup pushed